packages/editor/src/components/parts/program/configuration/ConfigurationPart.tsx

```diff
@@ -76,7 +76,7 @@ const ConfigurationPart = () => {
   return (
     <>
       {editorItems.map((widget, index) => (
-        <div key={index}>{renderWidgetComponent(widget)}</div>
+        <div key={index}>{renderWidgetComponent(widget, index.toString())}</div>

```

Comment:
not sure that this is correct. 
As far as I understood this, you should now have a id on the widget, which is corresponding to the key of the config data structure